email-alert-api/Makefile

```diff
@@ -0,0 +1,4 @@
+email-alert-api: ../email-alert-api
+	bin/govuk-docker run email-alert-api-default bundle
+	bin/govuk-docker run email-alert-api-default sh -c 'rake db:migrate 2>/dev/null || rake db:setup'
+	bin/govuk-docker run email-alert-api-default bundle exec rake
```

Comment:
I don't think we want rake here as that would run the test suite. I think we can just remove that line as we just need the former 2.